### PR TITLE
Added field for code of conduct

### PR DESCRIPTION
**Pull request by pratyusha972**
For checking this additional field, the before_action check_date line in app/controllers/projects_controller.rb has to be commented (temporarily, as the deadline for project submission is over).
codeofconduct
codeofconduct2
The changes have been made in 
views/projects/new.html.slim,
views/projects/show.html.slim,
controllers/projects_controller.rb 
and db/schema.rb.
(I have added a new field named code_of_conduct into the projects table).

### CODE REVIEW

**Comment by carpodaster** [comment]
Hello @pratyusha972 - thanks for submitting a PR. There are a lot of additional files in this PR and before I can review this PR, I'd kindly ask you to remove all those files that are not related to the feature/improvement.

There are several files ending on `~` – probably backup files from your text editor. You also checked in files from `log/` and `tmp/` which should be ignored by the project's `.gitignore` file.

**Comment by pratyusha972** [comment]
Hello @carpodaster - Sorry for the inconvenience. I have removed all the unnecessary files and committed only the required files. Can you please review it and give your feedback.

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@ def update
       if @project.update_attributes(project_params)
         format.html { redirect_to projects_path, notice: 'Project was successfully updated.' }
       else
-        format.html { render action: :new }
+        format.html { render action: :new }\
```
Comment (carpodaster) [inline app/controllers/projects_controller.rb]: Can you remove the trailing bashslash please?

**Comment by carpodaster** [comment]
Hej @pratyusha972 – looks good! Just one tiny little backslash to be removed and then I'm gonna merge :shipit:

**Comment by pratyusha972** [comment]
Hello @carpodaster , Changes done. Thank you for reviewing! :)

**Comment by carpodaster** [comment]
Thank you @pratyusha972

**Pull request merged**